### ISSUE TITLE
Support for "validation split" for Orca estimator

### ISSUE THREAD

**Issue by xiejinglei** [intel-analytics/analytics-zoo#3454]
In the fit function of `Keras`, there is an argument ` validation_split` which allows users to split the input dataset into a training set and a validation set according to a specific ratio. For example,
```py
from tensorflow.keras.layers import Dense

model = Sequential()
model.add(Embedding(max_features, 32))
model.add(SimpleRNN(32))
model.add(Dense(1, activation='sigmoid'))

model.compile(optimizer='rmsprop', loss='binary_crossentropy', metrics=['acc'])
history = model.fit(input_train, y_train,
                    epochs=10,
                    batch_size=128,
                    validation_split=0.2)
```
In the code above, 80% of the input dataset becomes the training set and 20% becomes the validation set. Currently Orca estimator does not support this function.

**Comment by hkvision** [comment]
@yangw1234 Take a look to see if we can support this.

**Comment by yangw1234** [comment]
It is hard to support it if input is a `data_creator` because the output of a `data_creator` is a `tf.data.Dataset` which possibly has been repeated and shuffled.
